package.json

```diff
@@ -39,6 +41,8 @@
 	"devDependencies": {
 		"ava": "^2.4.0",
 		"strip-ansi": "^6.0.0",
+		"tsd": "^0.17.0",
+		"typescript": "^4.3.2",
```

Comment:
Why are you including `typescript`?

Reply:
XO seems to require it to run eslint. Try to remove it and run `xo`, you'll see that it complains about missing `typescript` dependency.

Reply:
Ah right. I forgot about https://github.com/xojs/xo/issues/555.